### PR TITLE
Bubble and Combo Charts

### PR DESCRIPTION
**Pull request by cristinapopescu**
Hi Wes,

I have created 2 charts : 
- One is the Bubble Chart that will display all the open opportunities by owner using the probability and a custom field (stage duration) that will display the number of days singe the stage hasn't changed. The bigger the bubble is , the bigger the stage duration is. I had try to avoid using a custom field, but I coudn't.
- The second one is a combo Chart that is using the account object and displays the amount of closed won opportunities against the total amount of opportunities of each account (excluding the closed lost ones).

I have added the account object and opportunity object to git.

Please let me know if there is anything that you'll like me to change.

Cristina

### CODE REVIEW

**Comment by wesnolte** [comment]
This looks great Cristina. A few questions and tweaks:
- Did you include the Account and Opportunity object because you've customised them?
- Tom made a pull request just before you and that's now included in this repo but not in yours. Please can you (reverse integrate)[http://msdn.microsoft.com/en-us/library/vstudio/ee782536.aspx] the new changes and issue another pull request. Ben knows how to do this.

**Comment by cristinapopescu** [comment]
Hi Wes,

I have included those 2 objects because I had to create some custom field
in order to display the values on the charts. But I just realised I
should've add only the new fields created.

I will ask Ben to help me with this tomorrow morning and let you know after
I do the pull request.

**Comment by cristinapopescu** [comment]
Hi Wes,

I have refreshed my repository and added Tom's map.

Please let me know if I need to do any other changes.

Thank you.

**Comment by wesnolte** [comment]
:+1: I can now automatically merge so you've successfully reverse integrated, nice work.

With regards to the custom fields, is there any chance you can rework your examples so that they only use standard fields? Again Ben has done this for one of his examples so he can help you out. I'm trying to keep the repo as light as possible so I'd like to keep any object customisation out if possible.

**Comment by cristinapopescu** [comment]
I will look into finding a different scenario for both charts in order to use only standard fields. I will get back to you as soon as I've got something.

**Comment by cristinapopescu** [comment]
Hi Wes,

I have managed to display the same data in the bubble chart usign standard salesforce functionality. There is a stage duration standard field on the opportunity field history that will calculate the numbers of days for each stage. I have updated my repo with the new changes.

Regarding the combo chart I will try to get it updated by the end of tomorrow. (hope to get a much more interesting scenario for the chat to be a bir more complex. I will keep you updated on this one)

File: README.md [hunk 1]
```diff
@@ -9,9 +9,19 @@ Inspired by a post by the [PM for Analytics](https://medium.com/p/c87517ab52b1)
 ##Pie Chart (Summary Report)
 ![Example Bar Chart](https://dl.dropboxusercontent.com/u/4151695/html/Analytics%20API/pie-chart.png)
 
+<<<<<<< HEAD
```
Comment (wesnolte) [inline README.md]: This line means that you need to resolve a conflict from a prior merge. Let me know if you need help. You can use a tool like SourceTree to manage git tasks including merging and conflicts: http://www.sourcetreeapp.com/

Comment (cristinapopescu) [inline README.md]: I have fixed the issue and push the changes. :)

From:  Wes notifications@github.com
Reply-To:  wesnolte/Analytics-API-Demos
<reply+p-7758656-0240727954f3520a40769259bef3c43b6d0e14e9-4818673@reply.gith
ub.com>
Date:  Tuesday, 19 November 2013 14:45
To:  wesnolte/Analytics-API-Demos Analytics-API-Demos@noreply.github.com
Cc:  Cristina Popescu cpopescu@tquila.com
Subject:  Re: [Analytics-API-Demos] Bubble and Combo Charts (#7)

In README.md:

> @@ -9,9 +9,19 @@ Inspired by a post by the [PM for
> Analytics](https://medium.com/p/c87517ab52b1)
>  ##Pie Chart (Summary Report)
>  Example Bar 
> Chart
> 
> +<<<<<<< HEAD
> This line means that you need to resolve a conflict from a prior merge. Let
> me know if you need help. You can use a tool like SourceTree to manage git
> tasks including merging and conflicts: http://www.sourcetreeapp.com/

‹
Reply to this email directly or view it on GitHub
https://github.com/wesnolte/Analytics-API-Demos/pull/7/files#r7758656 .

**Comment by cristinapopescu** [comment]
Morning Wes,

I have managed to diplay the data in the combo chart without using any custom fields. I have commited my changes.

Please let me know if there is anything else that needs to be changed.

Thank you.

**Comment by wesnolte** [comment]
Excellent. Last request, please remove the Account and Opportunity object from the commit. All orgs will have these objects so we don't need to add them in.

**Comment by wesnolte** [comment]
Checkout the "git rm" command.

**Comment by cristinapopescu** [comment]
Done! :)

**Comment by wesnolte** [comment]
:100:

**Pull request merged**